tests/contrib/mongoengine/test.py

```diff
@@ -70,10 +70,14 @@ def test_insert_update_delete_query(self):
         eq_(artists[0].first_name, 'Joni')
         eq_(artists[0].last_name, 'Mitchell')
 
+        # query names should be used in pymongo>3.1
+        import pymongo
```

Comment:
why do we do this import here instead of at the top?

Reply:
My reasoning was that the import only mattered for this test, but can move it up if this is practice we want to avoid.

Reply:
yeah, moving up would be good, since it also doesn't hurt anything by being up top, or the test doesn't require it to be in here (e.g. test that something happens on import).